src/main/java/com/epam/internship/carrental/car/CarServiceImpl.java

```diff
@@ -0,0 +1,234 @@
+package com.epam.internship.carrental.car;
+
+import com.epam.internship.carrental.car.enums.CarGearbox;
+import com.epam.internship.carrental.car.enums.CarType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+/**
+ * Implementation of the {@link com.epam.internship.carrental.car.CarService} interface.
+ */
+@Service
+@Qualifier("carService")
+public class CarServiceImpl implements CarService {
+
+    private static final String AUTH_HEADER = "Token ";
+    private static final String AUTH_TOKEN = AUTH_HEADER + "token";
+
+    private final CarRepository carRepository;
+
+    @Autowired
+    public CarServiceImpl(final CarRepository carRepository) {
+        this.carRepository = carRepository;
+    }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @param make       the make of the car
```

Comment:
If you have `{@inheritDoc}` then you don't need to specify the parameters either.